tfx/examples/bert_cola/bert_utils.py

```diff
@@ -0,0 +1,155 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+from typing import List, Text
+
+import absl
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_transform as tft
+import tensorflow_hub as hub
+from bert_tokenizer_utils import Special_Bert_Tokenizer
+from bert_models import BertForSingleSentenceSentimentAnalysis
+
+from tfx.components.trainer.executor import TrainerFnArgs
+
+_TRAIN_BATCH_SIZE = 64
```

Comment:
Lets set the batch sizes, learning rates, and epochs (i.e. training steps) according to what is suggested by the paper.

Reply:
will do